relay-quotas/src/quota.rs

```diff
@@ -94,12 +101,41 @@ impl From<MetricNamespace> for MetricNamespaceScoping {
     }
 }
 
-/// Data categorization and scoping information.
+/// An owned data categorization and scoping information.
 ///
-/// `ItemScoping` is always attached to a `Scope` and references it internally. It is a cheap,
-/// copyable type intended for the use with `RateLimits` and `RateLimiter`. It implements
-/// `Deref<Target = Scoping>` and `AsRef<Scoping>` for ease of use.
+/// [`OwnedItemScoping`] is similar to [`ItemScoping`] but owns its data rather than
+/// borrowing it. This makes it suitable for storing in data structures or passing
+/// between threads.
 #[derive(Clone, Copy, Debug)]
+pub struct OwnedItemScoping {
```

Comment:
This entire thing is `Copy`, why not just drop the reference from `ItemScoping`?

Reply:
I mistakenly marked the type as `Copy`.